src/fcp/v2_parser.py

```diff
@@ -88,7 +88,8 @@
     identifier: CNAME
     string: ESCAPED_STRING
     number: SIGNED_NUMBER
-    value : identifier | number | string
+    value : array | identifier | number | string
+    array : "[" [value ("," value)*] "]"
```

Comment:
learning: are the square brackets around `value ("," value)*` so that we can define empty arrays?